### PR TITLE
Update project-activity.md

### LINKED ISSUE #12928: Wrong Description
In the User Activity Table, the descriptions for "Projects visited (Web)" and "Tasks created (Web)" are the same -- they both talk about the number of tasks created.  It seems that the description for the first should be the number of projects visited.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 88b68222-3eac-0447-de7f-9b7ba70f1794
* Version Independent ID: 88b68222-3eac-0447-de7f-9b7ba70f1794
* Content: [Microsoft 365 admin center Project activity  - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/activity-reports/project-activity?view=o365-worldwide#interpret-the-project-activity-report)
* Content Source: [microsoft-365/admin/activity-reports/project-activity.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/activity-reports/project-activity.md)
* Service: **microsoft-365-business**
* GitHub Login: @camillepack
* Microsoft Alias: **camillepack**

### PR DESCRIPTION
**Pull request by msbemba**
Fixes Projects visited(web) description
Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12928

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/activity-reports/project-activity.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The following are definitions for each metric in the user activity table.
 |Display name|The full name of the user.|
 |Last activity date|The latest date the user in that row had activity in Project, including any of the activities in the summary reports.|
 |Projects visited (Desktop)|The number of projects opened by the user in the Project Online desktop client during the time range selected in the top right of the page.|
-|Projects visited (Web)| The number of tasks created by the user in Project for the Web during the time range selected in the top right of the page.|
+|Projects visited (Web)| The number of Projects visited by the user in Project for the Web during the time range selected in the top right of the page.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/project-activity.md]:
```suggestion
|Projects visited (Web)| The number of projects visited by the user in Project for the Web during the time range selected in the top right of the page.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy reciew.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@camillepack Please merge this PR. Thanks

**Pull request merged**